src/index.css

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap');
+
```

Comment:
- [ ] While implementing changes from the last review, you've created significant changes to the style features of the application. Please see below:

Your application now looks like the image below: See the highlighted section. Please, let's fix it.
![Screenshot 2021-09-15 at 18-32-27 React App](https://user-images.githubusercontent.com/70150423/133489866-6e83b593-ddf7-4805-9a64-aa1923264d76.png)

Reply:
In your dev tools is gonna look like that obviously, although this project doesn't require responsive design I tried to fix it with the size of their widths